sfge/src/test/java/com/salesforce/graph/ops/expander/ApexPathExpanderTest.java

```diff
@@ -76,6 +84,73 @@ public void testSingleton() {
         MatcherAssert.assertThat(path.getInvocableVertexToPaths().entrySet(), hasSize(equalTo(2)));
     }
 
+    @Test
+    public void testRegistryCleanup() {
```

Comment:
I verified experimentally that this test passes with my changes, and fails without them.